resources/dicts/events/misc/general.json

```diff
@@ -1,3722 +1,3943 @@
         "m_c": {
-          "age": ["kitten"],
-          "relationship_status": ["child/parent"]
+            "status": ["apprentice", "medicine cat apprentice", "mediator apprentice"],
+            "trait": [
+                "troublesome",
+                "fierce",
+                "childish",
+                "bold",
+                "daring",
+                "righteous",
+                "confident",
+                "adventurous",
+                "shameless",
+                "sneaky",
+                "strange",
+                "cunning",
+                "rebellious"
+            ]
         },
-        "r_c":{
-          "age":["any"],
-          "status": ["any"]
+        "other_clan": {
+            "changed": 2
         }
     },
     {
-        "event_id": "gen_misc_any_accessorykitparent2",
+        "event_id": "gen_misc_war_secret_friendship_apps_quiet",
         "location": ["any"],
         "season": ["any"],
-        "sub_type": ["accessory"],
-        "weight": 15,
-        "event_text": "m_c finds some acc_singular to wear in {PRONOUN/m_c/poss} fur to impress r_c!",
-        "new_accessory": ["WILD", "PLANT"],
+        "sub_type": ["war"],
+        "weight": 20,
+        "event_text": "m_c is discovered gambolling along the border with an apprentice from o_c_n! m_c's tail twitches as {PRONOUN/m_c/subject} {VERB/m_c/apologize/apologizes}, plucking up the courage to insist that {PRONOUN/m_c/poss} friend isn't like the rest of o_c_n at all! This war sucks.",
         "m_c": {
-            "age": ["kitten"],
-            "relationship_status": ["child/parent"],
-            "trait": ["attention-seeker", "charming"]
+            "status": ["apprentice", "medicine cat apprentice", "mediator apprentice"],
+            "trait": [
+                "charismatic",
+                "nervous",
+                "insecure",
+                "compassionate",
+                "thoughtful",
+                "calm",
+                "loving",
+                "wise",
+                "oblivious"
+            ]
         },
-        "r_c":{
-          "age":["any"],
-          "status": ["any"]
+        "other_clan": {
+            "changed": 2
         }
     },
     {
-        "event_id": "gen_misc_any_kitapprentice1",
+        "event_id": "gen_misc_war_omgpeaceprophecy",
         "location": ["any"],
         "season": ["any"],
-        "weight": 15,
-        "event_text": "m_c is scolded after sneaking out of camp.",
+        "sub_type": ["war"],
+        "weight": 20,
+        "event_text": "m_c is plagued by strange dreams where the leaders of o_c_n and c_n come together peacefully to trade fresh-kill.",
         "m_c": {
-            "age": ["kitten", "adolescent"]
+            "status": ["any"],
+            "age": ["kitten", "adolescent", "young adult", "adult", "senior adult", "senior"]
```

Comment:
a skill constraint here for the dream and prophet skills would be nice, could even make it the lowest skill threshold so that even kittens can hit it